CountlyViewTrackingInternal.m

```diff
@@ -348,6 +348,11 @@ - (void)stopViewWithIDInternal:(NSString *) viewKey customSegmentation:(NSDictio
         segmentation[kCountlyVTKeyName] = viewData.viewName;
         segmentation[kCountlyVTKeySegment] = CountlyDeviceInfo.osName;
         
+        if (viewData.segmentation)
```

Comment:
1.what will happen if someone calls 'stopViewWithName' or stopallviews or if its an autostoppedview? comes back to here?

Reply:
Yes, In all stop view cases this the central place.